packages/odf/utils/ocs.ts

```diff
@@ -136,16 +146,22 @@ export const getResourceProfileRequirements = (
  * @param profile A resource profile.
  * @param cpu The amount CPUs.
  * @param memory The amount of selected nodes' memory in GiB.
- * @param memory The amount of OSD pods.
+ * @param osdAmount The amount of OSD pods.
+ * @param isS390x Whether the platform is s390x.
  * @returns boolean
  */
 export const isResourceProfileAllowed = (
   profile: ResourceProfile,
   cpu: number,
   memory: number,
-  osdAmount: number
+  osdAmount: number,
+  isS390x: boolean = false
```

Comment:
Suggestion: instead of adding a flag param for a certain architecture (which eventually will lead to adding more & more flags for different architectures), let's pass the architecture and invoke `isS390xArchitecture` inside:
```suggestion
  architecture: string
```